src/components/Map.vue

```diff
@@ -1370,7 +1370,7 @@ export default {
       this.streamgageMarkers.clearLayers();
       this.toggleStreamgage(this.streamgageMarkers, this.currentZoom);
       this.nfhlLayer.remove();
-      this.getNfhlLayer();
+      this.toggleNfhl(this.nfhlLayer, this.currentZoom);
```

Comment:
I don't think you actually need to pass the `this.currentZoom` to the 'toggleNfhl` function. That function only has 1 argument which is the nfhlLayer you're passing in.

```suggestion
      this.toggleNfhl(this.nfhlLayer);
```

Reply:
so true